pkg/manager/impl/validation/execution_validator.go

```diff
@@ -157,3 +162,12 @@ func ValidateWorkflowExecutionIdentifier(identifier *core.WorkflowExecutionIdent
 	}
 	return nil
 }
+
+// Because single task executions don't use launch plans, some parameters that are optional overrides for conventional
+// ExecutionCreateRequests are actually mandatory.
+func validateLaunchSingleTaskExecutionReq(request admin.ExecutionCreateRequest) error {
+	if request.Spec.AuthRole == nil {
```

Comment:
Should this check inside the AuthRole as well? like `AuthRole.Method.....`

Reply:
excellent point, done